docs/quicktips/multi-cam.md

```diff
@@ -12,75 +12,86 @@ description: >-
 {{ page.description }}
 
 __Disclaimer__  
-Thank you [Charlie_Powell](https://community.octoprint.org/u/Charlie_Powell) for letting us adapt your guide for mainsailOS!  
+Thank you to [Charlie_Powell](https://community.octoprint.org/u/Charlie_Powell) for letting us adapt his guide for MainsailOS!  
 The original guide can be found [here](https://community.octoprint.org/t/setting-up-multiple-webcams-in-octopi-the-right-way/32669).
 {: .info}
 
 ## Prerequisites
-You need:
+For multicam to work, you will need:
 
 - mainsailOS v0.5.0 or higher
-- ssh access
-- at least two cameras
+- SSH access
+- Two or more cameras
 
 __WARNING__  
-**If you are working on a existing installation, make a backup!**
+**If you are working with an existing installation, make a backup!**
 {: .warning}
 
 ## Creating the configuration files
-mainsailOS supports an infinite number of webcam*.txt-style configuration files. They can be placed in
+MainsailOS supports an infinite number of `webcam*.txt` style configuration files. They should be placed in
 `/home/pi/klipper_config`.
 
-In the UI, got to 'Settings > Interface' click on (1) and save as `webcam2.txt`.
+In Mainsail's user interface, go to 'Settings > Interface' and click on (1) and save as `webcam2.txt`.
 
 :image:
 
-Now you should have two identical files:
+You should now have two identical files with different names:
 1. `/home/pi/klipper_config/webcam.txt`
 2. `/home/pi/klipper_config/webcam2.txt`
 
-## Editing the original configuration file for a specific camera
-Find out the path to the camera by ID. This makes it much easier to stop /dev/video0 and /dev/video1 from switching on you.
+## Edit the original configuration file for a specific camera
+Find the path to the camera by its ID. This makes it easier to stop `dev/video0` and `/dev/video1` from being switchedin the future.
 
-Run `ls /dev/v4l/by-id`, copy your camera id and paste it into the editor in mainsail. For example:
+In a termina, run the comman `ls /dev/v4l/by-id`, copy your camera id and paste it into the editor in Mainsail. For example:
 
 ```bash
 $ ls  /dev/v4l/by-id/
+```
+
+returns
+
+```
 usb-046d_0825_88C56B60-video-index0  usb-046d_0825_88C56B60-video-index1
 ```
-I'll use the first one `usb-046d_0825_88C56B60-video-index0`
+This example will use the first one: `usb-046d_0825_88C56B60-video-index0`
 
 
 __Note__  
 Not sure which device is which? You can run lsusb to match a name to an ID.
 {: .info}
 
-Then edit `/home/pi/klipper_config/webcam.txt` in by clicking on it in mainsail.
+Edit `/home/pi/klipper_config/webcam.txt`  by clicking on it in Mainsail.
 
-Set `camera="usb"` at the top, and the `camera_usb_options` line to indicate the device:
+Add `camera="usb"` at the beginning of the file and then add a line with  `camera_usb_options` to indicate the device:
```

Comment:
Is "add" the right term? these options are already in the file, but commented out:

https://github.com/mainsail-crew/MainsailOS/blob/f622e1ce6ebd3e6c69d632ebce4ce709669571bd/src/modules/mjpgstreamer/filesystem/home/pi/klipper_config/webcam.txt#L16
https://github.com/mainsail-crew/MainsailOS/blob/f622e1ce6ebd3e6c69d632ebce4ce709669571bd/src/modules/mjpgstreamer/filesystem/home/pi/klipper_config/webcam.txt#L24

Not sure about this one to be honest. :D